### PR TITLE
Sample add transform via aruco marker opencv contrib

### PR DESCRIPTION
**Pull request by SatjaSivcev**
This PR modifies the sample to use OpenCV contrib's ArUco library instead of ArUco nano library.

It also addresses some comments from PR: https://github.com/zivid/zivid-cpp-samples/pull/153

### CODE REVIEW

File: source/Applications/Advanced/TransformPointCloudViaArucoMarker/TransformPointCloudViaArucoMarker.cpp [hunk 1]
```diff
@@ -36,32 +37,13 @@ namespace
...
-        auto bgr = cv::Mat(pointCloud.height(), pointCloud.width(), CV_8UC4); // NOLINT(hicpp-signed-bitwise)
-        cv::cvtColor(rgb, bgr, cv::COLOR_RGBA2BGR);
-
-        return bgr;
-    }
-
-    std::vector<cv::Point2f> estimate2DMarkerCenters(const aruconano::Marker &marker)
+    cv::Point2f estimate2DMarkerCenter(const int &markerId,
+                                                     std::vector<cv::Point2f> &markerCorners)
     {
```
Comment (chrisasc) [inline source/Applications/Advanced/TransformPointCloudViaArucoMarker/TransformPointCloudViaArucoMarker.cpp:42]: Add a check to see if marker was detected or not.

File: source/CMakeLists.txt [hunk 2]
```diff
@@ -92,7 +91,7 @@ else()
 endif()
 
 if(USE_OPENCV)
-    find_package(OpenCV 4.1.0 COMPONENTS core highgui calib3d)
```
Comment (chrisasc) [inline source/CMakeLists.txt:95]: change this back to 4.1.0

**Review by chrisasc — CHANGES_REQUESTED** [review]
Change the version of OpenCV back to 4.1.0 and its all good

File: source/CMakeLists.txt (hunk 2 above)
Comment (SatjaSivcev) [reply source/CMakeLists.txt:95]: done

**Pull request merged**